src/clj/ataru/applications/application_store.clj

```diff
@@ -155,6 +155,7 @@
   [answers fields]
   (filter (fn [field]
             (and (= "attachment" (:fieldType field))
+                 (not (get-in field [:params :hidden]))
```

Comment:
Tämä kohta kannattaa kirjoittaa näin:

```
(-> field :params :hidden not)
```

`get-in` on hidas, `->` -makro nopea. Ohessa pieni mielivaltainen vertailu `get-in` ja `->` välillä, joka antaa hyvän kuvan näiden välisestä suhteesta:

### get-in

```
(not (get-in
           [:lol {:bal {:foo {:bar true}}}]
           [:lol :bal :foo :bar]))
```

             Execution time mean : 106,015866 ns

### ->

```
(-> [:lol {:bal {:foo {:bar true}}}]
        :lol
        :bal
        :foo
        :bar
        not)
```

             Execution time mean : 3,029602 ns

Eli em. esimerkissä `get-in` toimii noin 34 kertaa hitaammin kuin `->`.